### DISCUSSION TITLE
Quick way to restore old task manager, top ribbon area of file explorer

### DISCUSSION THREAD

**Discussion by Arun72107** [valinet/ExplorerPatcher#1555] [Ideas]
Hi valinet

Please give options to restore classic task manager as it as in win 8.1/7

Also the top ribbon area in file explorer looks bad... I would like to get the old ribbon design as it was in win 8.1/10

Also the left navigation pane in file eexplorer'sgrouping design is really poor.. I love how everything was neatly organized in win 8.1

Option to bring back cool balloon notifications and improvements to win 8.1 start..ability to customize close,maximize minimize button to make it look like 8.1

I installed win 8.1 icon pack for win 11 but these are the features which I miss.. I really wish we soon reach a day in the feature where explorer patcher will be able to provide a single button click to get the True win 8.1 start screen, win 8.1 file explorer, windows theme styles..etc

Thanks

**Comment by pyrates999** [comment]
Windows 8.x was the worst OS Microsoft ever released.  But to each their own.

**Reply by ghost** [reply]
I'd rather have 8.1 than 11 Windows. I like Windows 10, 8.1 too, but 11 is a garbage design stolen from the Cupertino company, no experiments and something special.

**Reply by OrthodoxWindows** [reply]
In terms of customization, Windows 8 and better than Windows 10, itself better than Windows 11. For several reasons, I would prefer to use Windows 8.x than Windows 7, because it is possible to obtain the classic theme with DWM ( unlike Windows 7).

**Reply by pyrates999** [reply]
Windows 7 was the best OS IMO.

**Comment by elderlabs** [comment]
I'd suggest a feature in ExplorerPatcher that forces Task Manager to open as the following: `taskmgr -d`

This will open Task Manager with the superior Windows 10 theme, though for dark mode users, it will lack a dark theme.

There are alternative dark themes to work around this, but /shrug.

**Reply by pyrates999** [reply]
That is beyond what EP is for I think.

**Reply by Amrsatrio** [reply]
I can do this for the taskbar right click. But not Ctrl+Shift+Esc.

**Reply by ghost** [reply]
> That is beyond what EP is for I think.

And I thought the main goal of EP was to get the perfect user experience by bringing back the most successful features from previous versions of Windows.
Personally I think that would be awesome, especially if it's called via ctrl+alt+delete. Even though the old Task Manager only has a white layout, it's much faster, requires fewer clicks to operate, and doesn't have that ugly side-by-side arrangement of items as if it's dirt like Ubuntu. 
You can bring back the spiffy start and taskbar from Windows 10, so why can't Task Manager be brought back. I've personally already tried this and it works, I see no reason NOT to implement it.

**Reply by pyrates999** [reply]
Oh I misunderstood what you were asking for.  I thought you meant to add a `-d` option when you want to manually run `taskmgr` from the command line.
